SPECS/perl-Bytes-Random-Secure/perl-Bytes-Random-Secure.spec

```diff
@@ -0,0 +1,166 @@
+%global __requires_exclude %{?__requires_exclude:%{__requires_exclude}|}^perl\\((MIME::QuotedPrint|Scalar::Util)\\)$
+# Perform optional tests
+%bcond_with perl_Bytes_Random_Secure_enables_optional_test
+
+Summary:        Perl extension to generate cryptographically-secure random bytes
+Name:           perl-Bytes-Random-Secure
+Version:        0.29
```

Comment:
note sure it matters but that perl code seems to be no longer supported (no change since 2015)

-> https://github.com/daoswald/Bytes-Random-Secure
-> https://metacpan.org/dist/Bytes-Random-Secure

Reply:
Thanks for the heads-up! I think for now we have to rely on these projects, because other Perl dependencies added here need them. It's not trivial (possible?) to remove them at this point.